gateway/modules/crud/crud.go

```diff
@@ -57,7 +56,7 @@ type Crud interface {
 	RawQuery(ctx context.Context, query string, args []interface{}) (int64, interface{}, error)
 	GetCollections(ctx context.Context, project string) ([]utils.DatabaseCollections, error)
 	DeleteCollection(ctx context.Context, project, col string) error
```

Comment:
We can remove the project parameter from all methods since nobody is using it

Reply:
We can also remove it from all the exposed method in `operations.go` and `internal.go`